pkg/stash/with_redis.go

```diff
@@ -4,55 +4,92 @@ import (
 	"context"
 	"time"
 
-	lock "github.com/bsm/redislock"
-	"github.com/go-redis/redis/v7"
+	"github.com/bsm/redislock"
+	"github.com/go-redis/redis/v8"
 	"github.com/gomods/athens/pkg/config"
 	"github.com/gomods/athens/pkg/errors"
 	"github.com/gomods/athens/pkg/observ"
 	"github.com/gomods/athens/pkg/storage"
 )
 
+const (
```

Comment:
We should not need these anymore as the config should pass in the default value right, can we throw an error if the config is not passing default values.

Reply:
Yea makes sense, will update this.

Reply:
Updated.